api/app/src/main/kotlin/packit/controllers/ConfigurationController.kt

```diff
@@ -0,0 +1,21 @@
+package packit.controllers
+
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import packit.AppConfig
+import packit.model.dto.LogoDto
+
+@RestController
+@RequestMapping("/configuration")
```

Comment:
I think we want to try to be consistent between the existing `auth/config` endpoint and this. Perhaps this gives the wrong impression that all fetchable configuration will be under the `configuration` route whereas we've already got a competing pattern of `[FEATURE TYPE]/config` endpoints. So perhaps this endpoint should be `logo/config` with corresponding inversion of controller name and method.

Reply:
I'll rename these routes and controllers to 'branding configuration'.